XRTK-Core/Assets/XRTK/Inspectors/PropertyDrawers/TypeReferencePropertyDrawer.cs

```diff
@@ -330,11 +328,11 @@ private static Type[] FindTypesByName(string typeName, SystemTypeAttribute filte
             var types = new List<Type>();
             var filteredTypes = GetFilteredTypes(filter);
 
-            for (var i = 0; i < filteredTypes.Count; i++)
+            foreach (var type in filteredTypes)
```

Comment:
Why change a performant FOR loop for a more expensive ForEach?

Reply:
I'm not worried about the for each loops in editor code.